src/DIRAC/RequestManagementSystem/DB/RequestDB.py

```diff
@@ -600,7 +648,7 @@ def getRequestSummaryWeb(self, selectDict, sortList, startItem, maxItems):
             "RequestID",
             "RequestName",
             "JobID",
-            "OwnerDN",
+            "Owner",
```

Comment:
I guess you need a corresponding PR for the webapp then ?

Reply:
https://github.com/DIRACGrid/WebAppDIRAC/pull/714